src/app/Providers/components/VMwareProviderHostsTable/SelectNetworkModal.tsx

```diff
@@ -11,21 +11,13 @@ interface ISelectNetworkModalProps {
   onClose: () => void;
 }
 
-// TODO replace the mock data with real data
-const MOCK_NETWORKS: IHostNetwork[] = [
-  { name: 'storage_network', address: '192.168.0.1/24' },
-  { name: 'compute_network', address: '192.168.0.2/24' },
-  { name: 'other_network', address: '192.168.0.3/24' },
-];
-
 const SelectNetworkModal: React.FunctionComponent<ISelectNetworkModalProps> = ({
   selectedHosts,
   onClose,
 }: ISelectNetworkModalProps) => {
-  console.log('TODO: generate options for selected hosts: ', selectedHosts);
-  const networkOptions = MOCK_NETWORKS.map((network) => ({
-    toString: () => formatHostNetwork(network),
-    value: network.name,
+  const networkOptions = selectedHosts.map((host) => ({
+    toString: () => `${formatHostNetwork(host.network)} - ${host.bandwidth} - ${host.mtu}`,
+    value: host.network.name,
```

Comment:
@gildub to display the bandwidth/MTU as an option description like the mockups show, you can add `props: { description: '...' }` to these option objects. Any props available on the PatternFly `<SelectOption>` can be passed this way, it's an optional property of our `OptionWithValue` interface in SimpleSelect.

Reply:
@vconzola, yes I did have an issue having those descriptions on a separate line. @mturley thanks for the pointers.